lib/formatters/table.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+
+var chalk,
+    table,
+    pluralize,
+    drawReport,
+    drawTable;
+
+chalk = require("chalk");
+table = require("table").default;
+pluralize = require("pluralize");
+
+drawReport = function(results) {
```

Comment:
It is ironic that ESLint codebase enforces code conventions that are not enforced programmatically.